.github/workflows/ci-test-reporter.yml

```diff
@@ -0,0 +1,20 @@
+---
+## Workflow to process the JUnit test results and add a report to the checks.
+name: ci-test-reporter
```

Comment:
Out of curiosity, what's the reason for using a different naming convention? we've been using the name `test-reporter` instead

Reply:
The main reason was because in https://github.com/elastic/synthetics/pull/730 there is another workflow that also has needs a test reporter.

I was thinking about merging it into one in a follow-up. But that would required a little bit of dynamicity I guess. But I already had something in mind.

Reply:
Or can I subscribe to multiple workflows in a single test reporter without any unwanted sideeffects?

Reply:
It's possible to add as many workflows as needed as long as the name of the artifacts are the same.

Though I think I understand now your concern, if there are multiple test jobs/workflows then listing the holistic overview of all of them in one single place won't be possible (at the moment) hence splitting the test-report in different and specialised ones will help with knowing the test results for each test workflow. Or maybe I make up the above :)

Reply:
Although in this particular case, tests are running as jobs within the same workflow, hence I'd expect there will be no issues if and only if the artifacts to be stored don't clash and have an unique name.

Reply:
Yes, this was kinda my concern. Splitting it was to be on the safe side.
Also the paths are slightly different for each workflow. Of course I can do a more generic glob pattern.

Reply:
But I got your point. Let me rename it again. 

Then if it's merged I will be able to see the output on the other PR.

Reply:
Let me know when it's renamed it 🙏

Reply:
https://github.com/elastic/synthetics/pull/728/commits/c95167cecac7d0b0c5efd05b9d5fd25185442225